pkg/logentry/stages/pipeline.go

```diff
@@ -108,14 +114,22 @@ func (p *Pipeline) Name() string {
 
 // Wrap implements EntryMiddleware
 func (p *Pipeline) Wrap(next api.EntryHandler) api.EntryHandler {
+	// start flushing every 100ms
+	go func() {
+		for range time.Tick(100 * time.Millisecond) {
```

Comment:
This needs a way to be stopped. You could use a context passed via the construct of the Pipeline.

Reply:
or a close function that will cancel a context, I would still make this context from the original context so the cancellation above this context is propagated here.